README.md

```diff
@@ -39,12 +39,12 @@ end
 Set your different credentials in the file by uncommenting the lines and adding your keys.
 
 ------------------
-If you are using devise_token_auth gem run this generator.
+If you are using devise_token_auth gem, run this generator.
 
     $ rails g apple_sign_in:appple_sign_in_controller [scope]
 
-In the scope yo need to write your path from controllers to yours existents devise controllers.
-An example '$ rails g apple_sign_in:appple_sign_in_controller api/v1/'
+In the scope you need to write your path from controllers to yours existent devise controllers.
+An example `$ rails g apple_sign_in:appple_sign_in_controller api/v1/`
 This will generate a new controller: `controllers/api/v1/apple_sign_in_controller.rb`.
```

Comment:
maybe in the future it'd be good to rename the generator and the controller to be apple auth instead of apple sign in?

Reply:
good catch,  I will open an issue